client/src/components/Main/index.js

```diff
@@ -74,13 +75,13 @@ const Main = () => {
 
   const redirectToEventView = (event) => {
     let eventID = event.eventID;
-    history.push(`/event?id=${eventID}`);
+    history.push(`/event?id=${eventID}&color=${ColorService.formatRawHex(event.color)}`);
   };
 
   const redirectToEventEdit = (event) => {
     let eventID = event?.eventID;
     if (eventID) {
-      history.push(`/event-edit?id=${eventID}`);
+      history.push(`/event-edit?id=${eventID}&color=${ColorService.formatRawHex(event.color)}`);
     } else {
       history.push(`/event-edit`);
```

Comment:
Since we don't have a way of editing an event's color, you could use the random colors here, and pass in a random one so that new events won't always just be the theme color.

Reply:
I'm going to leave this one as is; this would require more work to make the color cards on the main page match up with the colors on the event-edit page